src/Particles/pointsCloudSystem.ts

```diff
@@ -0,0 +1,1050 @@
+import { IndicesArray, FloatArray } from "../types";
+import { Color4, Color3 } from "../Maths/math";
+import { Vector2, Vector3, Vector4, TmpVectors, Matrix } from "../Maths/math.vector";
+import { Logger } from "../Misc/logger";
+import { VertexBuffer } from "../Meshes/buffer";
+import { VertexData } from "../Meshes/mesh.vertexData";
+import { Mesh } from "../Meshes/mesh";
+import { EngineStore } from "../Engines/engineStore";
+import { Scene, IDisposable } from "../scene";
+import { CloudPoint, PointsGroup } from "./cloudPoint";
+import { BoundingInfo } from "../Culling/boundingInfo";
+import { Ray } from "../Culling/ray";
+import { PickingInfo } from "../Collisions/pickingInfo";
+import { StandardMaterial } from "../Materials/standardMaterial";
+import { Texture } from "./../Materials/Textures/texture";
+import { Scalar } from "../Maths/math.scalar";
+
+/** Defines the 3 color options */
+export class PointColor {

```

Comment:
Will sort out tomorrow.

Reply:
Got stuck here and have left as is - need advice.

Tried changing to 

```
export const enum PointColor {
    /** color value */
    COLOR = 2,
    /** uv value */
    UV = 1,
    /** random value */
    RANDOM = 0,
    /** color value */
   STATED = 3
}
```
and then in any app once pcs a point cloud system is created I had to write , for example,

```
pcs.addSurfacePoints(box, 10000, 2);
```

rather than

```
pcs.addSurfacePoints(box, 10000, BABYLON.PointColor.COLOR);
```

Reply:
You should be able to do it like GUI for instance:
https://github.com/BabylonJS/Babylon.js/blob/master/gui/src/2D/controls/textBlock.ts#L10

Use:
https://www.babylonjs-playground.com/#XCPP9Y#1438 (line #37)

Reply:
Done and it works